### DISCUSSION TITLE
diagramme d'Ishikawa

### DISCUSSION THREAD

**Discussion by jpv-1** [freeplane/freeplane#889] [Ideas]
Bonjour,
could you add other plans like Ishikawa.
(if somebody has an idea, a template then it's possible as with x-m...)
Thanks/merci

**Comment by euu2021** [comment]
Take a look at #808 

If you think this kind of layout is useful, please comment at #892, so it can be improved in Freeplane. 
For example: How many levels do you usually use when you use this kind of diagram?

**Comment by jpv-1** [comment]
Hy and Thanks
I need (idem #808)  mostly a particular layout (ishikawa, fishbone).(as  #892 Mindmapper icons: 5eme Right)
Draw an axis of the problems in the middle (pb at right or left, as possible)  with the causes on both sides .
I am a teacher and my students will not go beyond 3 or 4 levels, but others may need more. https://fr.wikipedia.org/wiki/Fichier:Ishikawa_Fishbone_Diagram_cz.svg
The template frame when i create a new maps is not very ergonomic : big screen with thumbnails and one of theme with fishbone?
